test/bindings/each.spec.js

```diff
@@ -309,6 +311,23 @@ describe('each bindings', () => {
     el.unmount()
   })
 
+
+  it('Each bindings can use readonly field names from scope as itemName (issue https://github.com/riot/riot/issues/2925)', () => {
```

Comment:
Since this test is always performed without being tied to a real component, I think it should be in the riot/riot repository.